public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, colorManipulator } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+export default function PubdashFooter() {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.footer}>
+      <span className={styles.logoText}>
+        <a href="https://grafana.com/" target="blank">
```

Comment:
```suggestion
        <a href="https://grafana.com/" target="_blank"  rel="noreferrer noopener">
```
Even though it's a Grafana com URL, better to have `rel` for all external URL.